FutureCollars kurs/src/main/java/com/futurecollars/lesson5/task1/PalindromeChecker.java

```diff
@@ -0,0 +1,8 @@
+package com.futurecollars.lesson5.task1;
+
+public class PalindromeChecker {
```

Comment:
Dodatkowa klasa to trochę za dużo. Ta logika spokojnie mogła by być dodatkową metodą prywatną w klasie TextProcessor.